themes/bootstrap3/js/truncate.js

```diff
@@ -98,8 +99,16 @@ VuFind.register('truncate', function Truncate() {
       }
 
       if (shouldTruncate) {
-        var btnMore = '<button type="button" class="btn more-btn' + btnClass + '">' + moreLabel + ' <i class="fa fa-arrow-down" aria-hidden="true"></i></button>';
-        var btnLess = '<button type="button" class="btn less-btn' + btnClass + '">' + lessLabel + ' <i class="fa fa-arrow-up" aria-hidden="true"></i></button>';
+        var btnMore = '<button type="button" class="btn more-btn' + btnClass + '">'
+          + (settings['label-icons'] === 'before' ? '<i class="fa fa-arrow-down" aria-hidden="true"></i> ' : '')
```

Comment:
Should we define upIcon and downIcon variables so we don't have to repeat the icon definitions? In addition to reducing redundancy, I think that will make it easier to update this code in future when we roll out the icon helper everywhere.

Reply:
Makes sense, I'll revise.

Reply:
I've now made the icons part of the settings array.